### PR TITLE
Feature#33 login function

### LINKED ISSUE #33: Add login function to Administrators
# REQUIREMENTS
Create login function for Administrators validating email and password.
This validation will work on validate view before we can access to Administrator index view.

# Acceptance Criteria
- [x] Validate email and password
- [x] Message to wrong validation

### PR DESCRIPTION
**Pull request by SampabloC**
# Description
What did you implemented/Why did you implemented this:
 - Add login function to administrator using bcrypt gem.
 - What? Create session when administrator log in and destroy it when log out.
 -  Why? To avoid users not logged it modify restricted data.
 - Add an alert when log in is wrong for email or password.
 - Add personalize message when administrator log in to show a welcome message.
 
Resolve #33 
 
 # Testing
 - I add an administrator with email 'admin@admin.com' and password '1234', this is include on seed file.

 ## Screenshots 
Alert to wrong log in.
imagen

Welcome message to administrator.
imagen

### CODE REVIEW

**Review by AlejandroVI99 — DISMISSED** [review]
Nice!

**Review by SEVERO96 — DISMISSED** [review]

**Comment by SEVERO96** [comment]
Interesting gem, just out of curiosity, why didn't you use the devise gem? Blessings.

File: app/views/sessions/validation.html.erb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  </div>
+</div>
+<div>
+  <%= link_to "Back", root_path, class: "btn btn-info" %>
+</div>
+<% flash.each do |type, msg| %>
+  <div class="alert alert-info text-center">
+    <%= msg %>
+  </div>
+<% end %>
```
Comment (YilKanda) [inline app/views/sessions/validation.html.erb]: Add a blank line

File: app/views/administrators/check.html.erb [hunk 2]
```diff
@@ -4,5 +4,5 @@
     <input type="text">
     <button>Check In</button> <br>
 
-    <%= link_to "Log in", administrators_validation_path, class: "btn btn-primary" %>
+    <%= link_to "Log in", login_path, class: "btn btn-primary" %>
   </p>
```
Comment (YilKanda) [inline app/views/administrators/check.html.erb]: Add a blank line

File: db/seeds.rb [hunk 3]
```diff
@@ -6,12 +6,12 @@
...
-(1..50).each do |number|
-  employee_object = Employee.new(employee_number: number, name_employee: "employee_#{number}",
-                                 email_employee: "employee_#{number}@employee.com", position_employee: 'worker', attendance_date: '2021-07-09', administrator_id: 1)
-  employee_object.save
-end
+# (1..50).each do |number|
+#   employee_object = Employee.new(employee_number: number, name_employee: "employee_#{number}",
+#                                  email_employee: "employee_#{number}@employee.com", position_employee: 'worker', attendance_date: '2021-07-09', administrator_id: 1)
+#   employee_object.save
+# end
```
Comment (YilKanda) [inline db/seeds.rb]: If you don't occupy them, delete them.

**Review by YilKanda — CHANGES_REQUESTED** [review]
Remember that GitHub tells us when to leave a blank line (with a symbol of a circle with a red line in the middle.), and those codes that do not occupy can be eliminated (I mean the commented codes).

And if the visual part can be improved, it would be much better. (This can be considered for another issue)

**Review by AlejandroVI99 — APPROVED** [review]
LGTM JAVIER!

**Review by xFerchoVJ — APPROVED** [review]

**Review by martinmt-mx — APPROVED** [review]

**Review by SEVERO96 — APPROVED** [review]

**Review by YilKanda — APPROVED** [review]

**Pull request merged**